### DISCUSSION TITLE
URI encode '/' properly when clicking on tag

### DISCUSSION THREAD

**Discussion by lolcharacters** [civitai/civitai#193] [Ideas]
If you click on a tag with a '/' in it ([example page](https://civitai.com/models/4121/kda-all-out-ahri)), the URL will not be found because it is not encoded.

The page does exist for the example: https://civitai.com/tag/k%2Fda%20all%20out%20ahri

Looks the line is here: https://github.com/civitai/civitai/blob/c445e4f886dec98b69b9eb02e9a7d80840bbca01/src/pages/models/%5Bid%5D/%5B%5B...slug%5D%5D.tsx#L434
where it is using the tag directly. Instead it should be URI encoded. I think we only need to URI encode "/"? I believe browsers automatically URI encode other characters.

**Comment by JustMaier** [comment]
Thanks for the heads up on this. Looks like an easy fix. We'll get that resolved this week.

**Comment by manuelurenah** [comment]
Hey @lolcharacters! We have implemented a fix for this and should be out in our next release today. Thank you again for pointing it out 🙏
